staff/views.py

```diff
@@ -246,11 +267,12 @@ def make_announcement(request):
         form = MakeAnnouncementForm(request.POST)
         if form.is_valid():
             form.instance.created_by = DjangoUser.objects.get(id=request.user.id)
+            data = request.POST.copy()
```

Comment:
Why `copy()` it?

Reply:
Fixed in ffd66b3af3e9a66fe7b8673b0066ffb410ce6268